src/app/pages/payment/international-payment/international-payment.component.ts

```diff
@@ -0,0 +1,152 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import { Location } from '@angular/common';
+import {
+  FormGroup,
+  FormBuilder,
+  Validators,
+} from '@angular/forms';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { Subscription } from 'rxjs';
+import { SafeUrl, DomSanitizer } from '@angular/platform-browser';
+import { removeSubscription } from 'src/app/helpers/unsubscribe';
+import { PaymentService } from 'src/app/services/payment/payment-service';
+
+
+@Component({
+  selector: 'app-payments',
+  templateUrl: './international-payment.component.html',
+  styleUrls: ['./international-payment.component.scss']
+})
+export class InternationalPaymentComponent implements OnInit, OnDestroy {
+  loading: boolean;
+  payload: any;
+  form: FormGroup;
+  formData;
```

Comment:
`formData` is unused here. You can get rid of it if you arent using it

Reply:
@Tevinthuku done